ddtrace/internal/sma.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import division
+
+DEFAULT_SMA_WINDOW = 10
```

Comment:
I added a comment but I left it in the same file. We pass it as a default value to the `__init__()`. I thought it would be strange to have the default value in a different file. WDYT?

Reply:
hmm, would it make more sense to make the argument in `__init__()` required and then keep/pass this value in `AgentWriter`?